pom.xml

```diff
@@ -10,7 +10,7 @@
     </parent>
     <groupId>com.github.dblock</groupId>
     <artifactId>oshi-core</artifactId>
-    <version>1.3-SNAPSHOT</version>
+    <version>1.3</version>
```

Comment:
This is done via the maven release plugin so needs removed from this pull request.